src/__tests__/calculate.test.jsx

```diff
@@ -0,0 +1,38 @@
+import calculate from '../Logic/calculate';
```

Comment:
- [ ] Your test suites are failing, the first error is from your import statement. The 'L' in login is in lowercase. Kindly correct this please so your tests can pass.